### DISCUSSION TITLE
battery size

### DISCUSSION THREAD

**Discussion by steuerzentrale** [PricelessToolkit/MailBoxGuard#29] [Q&A] [answered]
sorry for my many questions, I hope its okay and you may could use my questions/inputs for your product. I bought a 14500 Li-Ion battery  https://www.nitecore.de/akkus/14500-li-ion-akkus/nl1485/nitecore-li-ion-akku-typ-14500-850mah-nl1485 which is unfortunately quiet long (thought the length is standardized) and it bends the battery contacts quiet a bit.

In the docs you point to these batteries: https://s.click.aliexpress.com/e/_DDtU0iT do you use flat head battery or pointed ones? I would like to buy the right ones in order to install the sensor permanently at its final site.

<img width="385" alt="Bildschirmfoto 2023-11-22 um 20 18 47" src="https://github.com/PricelessToolkit/MailBoxGuard/assets/75168550/d8553245-f2bd-4844-a6b2-442e40a14f78">
<img width="416" alt="Bildschirmfoto 2023-11-22 um 20 18 51" src="https://github.com/PricelessToolkit/MailBoxGuard/assets/75168550/071aecfb-faf0-4914-a3d2-830d96bf5f88">

**Comment by PricelessToolkit** [comment, marked as answer]
Hi, I am using the "pointed" type and yes it bends the battery contacts but not much. After installing the battery they are almost parallel. size is exactly like "AA" battery.

![batt](https://github.com/PricelessToolkit/MailBoxGuard/assets/81021972/abe38467-a28d-4b6c-b09b-7b60a21ddba9)

**Reply by steuerzentrale** [reply]
great, with my batteries I cant put the sensor in the 3d-printed case. thanks a lot
